.github/actions/deploy-template-operator-with-modulereleasemeta/action.yml

```diff
@@ -114,16 +117,17 @@ runs:
         popd
         make build-manifests
         echo "name: kyma-project.io/module/template-operator
-        channel: regular
         version: 1.0.1
         security: sec-scanners-config.yaml
-        defaultCR: ./config/samples/default-sample-cr.yaml
-        manifest: template-operator.yaml
-        annotations:
-          operator.kyma-project.io/doc-url: https://kyma-project.io" >> warning-module-config.yaml
-        modulectl create --module-config-file ./warning-module-config.yaml --registry http://localhost:5111 --insecure
+        defaultCR: https://github.com/kyma-project/template-operator/releases/download/1.0.1/default-sample-cr.yaml
```

Comment:
I expect the defaultCR also coming from localhost which aligned with manifest data, to have consistency here.